src/metaschema/oscal_implementation-common_metaschema.xml

```diff
@@ -192,10 +193,11 @@
                         <enum value="isa-title">Title of the Interconnection Security Agreement (ISA).</enum>
                         <enum value="isa-date">Date of the Interconnection Security Agreement (ISA).</enum>
                         <enum value="isa-remote-system-name">The name of the remote interconnected system.</enum>
-                        <enum value="ipv4-address">An Internet Protocol Version 4 interconnection address</enum>
-                        <enum value="ipv6-address">An Internet Protocol Version 6 interconnection address</enum>
-                        <enum value="direction">An Internet Protocol Version 6 interconnection address</enum>
+            &allowed-values-property-name-networked-components;
                   </allowed-values>
+                  <allowed-values target="[@type=('service', 'software')]/prop[has-oscal-namespace('http://csrc.nist.gov/ns/oscal')]/@name">
```

Comment:
Per discussion in #1913 and refining the issue as posed by the comment in https://github.com/usnistgov/OSCAL/issues/1913#issuecomment-2484136304, I am not sure why leveraged `system` typed components would not have similar need for like `software` and `service`, but I wanted to leave that open to interpretation here during the PR review.

```suggestion
                  <allowed-values target="[@type=('service', 'software', 'system')]/prop[has-oscal-namespace('http://csrc.nist.gov/ns/oscal')]/@name">
```